src/helpers/factory/article.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Factory } from 'rosie';
+
+export default Factory.define('article')
```

Comment:
use `chance` to generate content

Reply:
I am not the one who worked on  the article so, I think I should not change it to my commit but I can still do it if there is no problem, thank you

Reply:
This has been implemented